docs/span-api.asciidoc

```diff
@@ -147,10 +147,12 @@ This enables log correlation to APM traces with structured loggers.
 }
 ----
 
+
 [[span-to-string]]
-==== `span.toString()`
+==== `span.toString()` (deprecated)
```

Comment:
FWIW, our build supports a deprecated badge:

```
deprecated:[3.23.0]
```

It doesn't look very nice in titles, and I'm fine with how you've deprecated this, just figured it was worth mentioning in case you're unaware.
<img width="783" alt="Screen Shot 2021-09-23 at 2 26 59 PM" src="https://user-images.githubusercontent.com/5618806/134586632-18233230-d12d-44e1-9400-7564506aa84a.png">

Reply:
Not great in titles, but more informative. I'll take it. commit 328be4c0